tvx/exec_lotus.go

```diff
@@ -77,16 +78,25 @@ func runExecLotus(_ *cli.Context) error {
 
 		fmt.Println("roots: ", header.Roots)
 
-		fmt.Println("decoding message")
-		msg, err := types.DecodeMessage(tv.ApplyMessage)
-		if err != nil {
-			return err
-		}
-
 		driver := lotus.NewDriver(ctx)
 
-		fmt.Println("executing message")
-		spew.Dump(driver.ExecuteMessage(msg, header.Roots[0], bs, epoch))
+		root := header.Roots[0]
+
+		for i, m := range tv.ApplyMessages {
+			fmt.Printf("decoding message %v\n", i)
```

Comment:
prefer log or printing to stderr. here and below

Reply:
Let's address as a follow up, as we have many println and spew all over the codebase.